libkbfs/block_retrieval_queue.go

```diff
@@ -223,11 +235,26 @@ func (brq *blockRetrievalQueue) WorkOnRequest() <-chan *blockRetrieval {
 	return ch
 }
 
+func (brq *blockRetrievalQueue) triggerPrefetchAfterBlockRetrieved(
+	block Block, kmd KeyMetadata, priority int, hasPrefetched bool) {
+	if hasPrefetched {
```

Comment:
I did it this way because the alternative was to perform the check before each time this function is called. I could do that too, but it seemed less DRY.